app/main.py

```diff
@@ -11,35 +11,45 @@ def __str__(self) -> str:
     def __repr__(self) -> str:
         return f"Distance(km={self.km})"
 
-    def __add__(self, other: [int, float, Distance]) -> Distance:
+    def __add__(self, other: int | float | Distance) -> Distance:
         if isinstance(other, Distance):
             return Distance(self.km + other.km)
         return Distance(self.km + other)
 
-    def __iadd__(self, other: [int, float, Distance]) -> Distance:
+    def __iadd__(self, other: int | float | Distance) -> Distance:
         if isinstance(other, Distance):
             self.km += other.km
         else:
             self.km += other
         return self
 
-    def __mul__(self, other: [int, float, Distance]) -> Distance:
+    def __mul__(self, other: int | float | Distance) -> Distance:
         return Distance(self.km * other)
 
-    def __truediv__(self, other: [int, float, Distance]) -> Distance:
+    def __truediv__(self, other: int | float | Distance) -> Distance:
         return Distance(round(self.km / other, 2))
```

Comment:
btw, `__mul__ ` and `__truediv__ ` methods can't have a `Distance` type of `other` variable

Reply:
done